docs/sdks/server-sdks/dotnet/initialization.mdx

```diff
@@ -0,0 +1,133 @@
+---
+title: Initialization
+sidebar_position: 3
+---
+
+import ApiOptionRef from '@site/src/components/ApiOptionRef';
+
+The Eppo .NET SDK is easy to initialize while offering robust customization options, making it adaptable to various use cases such as custom caching requirements and ultra-low-latency initialization.
+
+## Initialize the SDK
+
+To complete basic initialization, you only need to provide an SDK key. [Create an SDK key](/sdks/sdk-keys) if you don't already have one. 
+
+```csharp
+var eppoClientConfig = new EppoClientConfig("<SDK_KEY>", null);
+var eppoClient = EppoClient.Init(eppoClientConfig);
+```
+
+## Use the SDK instance
+After initialization, you can use the SDK instance to assign a variation to a subject using the `Get*Assignment` methods.
+
+```csharp
+// Get the current user from your application context
+var user = GetCurrentUser();
+
+// Make an assignment
+var variation = eppoClient.GetStringAssignment(
+    "my-feature",
+    user.Id,
+    user.Attributes,
+    "default"
+);
+```
+
+## Advanced Configuration
+Basic initialization is great for most use cases, but the SDK provides options that you can use during initialization to customize the behavior of the SDK.
+
+### Initialization Options
+
+How the SDK fetches, serves, and caches experiment configurations is configurable via the `EppoClientConfig` class:
+
+<ApiOptionRef 
```

Comment:
Also supports setting the polling jitter:
https://github.com/Eppo-exp/dot-net-server-sdk/blob/89f65bb5779a7293beff878534381dcc871b03cd/dot-net-sdk/EppoClientConfig.cs#L38

Reply:
Adding that option. Making sure my search-fu is good and I understood all of this:

- Jitter is there to add some randomized additional time onto the polling frequency for reasons that are out of scope to explain in this.
- The default value per the consts file is 5000 ms

Reply:
correct